lib/ansible/runner/credential/openstack_credential.rb

```diff
@@ -0,0 +1,44 @@
+module Ansible
+  class Runner
+    class OpenstackCredential < Credential
+      def self.auth_type
+        "ManageIQ::Providers::EmbeddedAnsible::AutomationManager::OpenstackCredential"
+      end
+
+      # Modeled off of openstack injectors for awx:
+      #
+      #   https://github.com/ansible/awx/blob/1242ee2b/awx/main/models/credential/injectors.py#L70-L96
+      #
+      def env_vars
+        write_credentials_file
+        { "OS_CLIENT_CONFIG_FILE" => os_credentials_file }
+      end
+
+      def write_credentials_file
+        openstack_data = {
+          "clouds" => {
+            "devstack" => {
+              "verify" => false, # TODO:  NickL:  Should we implement this?
+              "auth"   => {
+                "auth_url"     => auth.host || "",
+                "username"     => auth.userid || "",
+                "password"     => auth.password || "",
+                "project_name" => auth.project || "",
+                "domain"       => auth.domain
+              }.delete_nils
```

Comment:
Same here, should this be `delete_blanks`?

Reply:
This one, however, was on purpose since `openstack` was using an if check around `"domain_name"`:

https://github.com/ansible/awx/blob/devel/awx/main/models/credential/injectors.py#L75-L76

which would have removed this key in practice (not sure why they still have a `default=''` there...).

That said, I f---ed up the key name, so I will go and fix that...

Reply:
Fixed the key.